src/test/groovy/tests/HealthCheckTest.groovy

```diff
@@ -1,37 +1,26 @@
 package tests
 
-import io.restassured.RestAssured
 import io.restassured.path.json.JsonPath
 import io.restassured.response.Response
 import org.hamcrest.Matchers
+import org.testng.Assert
 import org.testng.Reporter
 import org.testng.annotations.BeforeSuite
 import org.testng.annotations.Test
 import steps.RepositorySteps
-import org.yaml.snakeyaml.Yaml
-import utils.Shell
 
 class HealthCheckTest extends RepositorySteps{
-    Yaml yaml = new Yaml()
-    def configFile = new File("./src/test/resources/testenv.yaml")
-    def config = yaml.load(configFile.text)
-    def artifactoryBaseURL
-    def artifactoryPingURL
-    def protocol
-
+    def artifactoryURL
 
     @BeforeSuite(groups=["common"])
     def setUp() {
-        protocol = config.artifactory.protocol
-        artifactoryBaseURL = "${protocol}${config.artifactory.external_ip}/artifactory"
-        artifactoryPingURL = "${protocol}${config.artifactory.external_ip}"
+        artifactoryURL = "${artifactoryBaseURL}/artifactory"
```

Comment:
You can just set this at line 13 and get rid of the whole function - it's a constant for the life of the class